src/aideck/loaders/config.py

```
@@ -0,0 +1,382 @@
+    }
+    
+    @classmethod
+    def create_config(
+        cls, 
+        loader_type: str, 
+        **kwargs
+    ) -> LoaderConfig:
+        """Create a configuration for the specified loader type."""
+        config_class = cls._config_map.get(loader_type.lower())
+        if not config_class:
+            raise ValueError(f"Unknown loader type: {loader_type}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)